api_generator/Cargo.toml

```diff
@@ -12,24 +12,24 @@ license = "Apache-2.0"
 anyhow = "1.0"
 array_tool = "1.0.3"
 dialoguer = "0.10.2"
-flate2 = "~1"
-globset = "~0.4"
+flate2 = "1"
+globset = "0.4"
 Inflector = "0.11.4"
 itertools = "0.11.0"
 lazy_static = "1.4.0"
 log = "0.4.8"
 path-slash = "0.2.1"
-proc-macro2 = "~1.0"
-quote = "~1.0"
+proc-macro2 = "1.0"
+quote = "1.0"
```

Comment:
These widen from `1.0.*` to `1.*.*`, probably okay because semver, and also currently proc-macro2/quote hasn't hit `1.1` yet